src/epd_driver/include/epd_driver.h

```diff
@@ -25,6 +25,13 @@ typedef struct {
   int height;
 } EpdRect;
 
+// Simple x and y coordinate
```

Comment:
Since this type is only used internally, it should remain in the C file and not be publicly exposed.